### ISSUE TITLE
Border outline of "Icons" in the Menu list items are not following the High Contrast standards on mouse hovering

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#348]
**Repro Steps**
* Set theme to High Contrast Black or White.
* Launch Calculator app.
* Navigate to "Open navigation" button and activate it.
* Hover the mouse on the menu list items in High Contrast Black/White theme.
* Observe the issue.

**Actual Result**
Border outline of "Icons" in the Menu list items are not following the High Contrast standards on mouse hovering.

**Expected Result**
Border outline of "Icons" in the Menu list items should follow the High Contrast standards on mouse hovering.

![image](https://user-images.githubusercontent.com/1414423/54739401-34c39300-4b75-11e9-8650-f2cf774c8be1.png)

**Test Environment**
OS Version: Windows 10 Build 18351.1
App Name: Calculator 
App Version: 10.1902.16.0

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.

**Comment by jeffsieu** [comment]
Can I work on this issue?

**Comment by grochocki** [comment]
Yes! Thanks for taking a look, @jeffsieu.

**Comment by kaceka** [comment]
It looks that this issue is already fixed.

![tmp](https://user-images.githubusercontent.com/12201900/95022927-55c9d700-067a-11eb-8f36-dee8cd710dfb.JPG)

**Comment by mcooley** [comment]
@kaceka thanks! It does indeed look like this is resolved in Calculator 10.2009.4.0.
